### ISSUE TITLE
Duplicate entries in the footer

### ISSUE THREAD

**Issue by nank1ro** [jesseduffield/lazygit#3612]
**Describe the bug**
The footer has duplicated entries, see screenshot below.
This bug happens in all the panels

**To Reproduce**

1. Open lazygit
2. See duplicate entries

**Expected behavior**
No duplicate entries

**Screenshots**
<img width="1678" alt="Screenshot 2024-05-29 at 15 36 14" src="https://github.com/jesseduffield/lazygit/assets/60045235/537978b8-2035-48a5-9f3d-644c0f9270a9">

**Version info:**
lazygit version
```
commit=3675570a391b1a49ddd198b4c7e71e17701d4404, build date=2024-03-23T09:09:11Z, build source=binaryRelease, version=0.41.0, os=darwin, arch=arm64, git version=2.39.3 (Apple Git-146)
```
git version
```
git version 2.39.3 (Apple Git-146)
```

**Comment by stefanhaller** [comment]
Interesting, can't reproduce, and I have no idea why this should happen on your system but not on mine.

I'm not even sure what I could ask you to get more information about this.

@jesseduffield Any ideas?

**Comment by jesseduffield** [comment]
can't reproduce on my end either, and also don't know why it would happen on only some people's computers.

**Comment by nank1ro** [comment]
If I have time this weekend I can give a look at the source code and try to find the reason, having the machine with the error.
Can you guide me on which files to check for these entries?

**Comment by stefanhaller** [comment]
The function that renders the keybindings is here: https://github.com/jesseduffield/lazygit/blob/7492521829114b935c91a4eab856a554160122bd/pkg/gui/options_map.go#L35

The `bindingsToDisplay :=` a few lines down is where it filters down the list to find the ones to display, but that's probably not the issue. It looks like for you the whole list of keybindings is duplicated for some reason, so we need to find out why that happens.

**Comment by giannisp** [comment]
I have just noticed this issue as well.
It's happening while using the panel "[1] - Status" and switching to recent repos.
Open lazygit with repo A, switch to repo B, switch back to repo A, options items are doubled.

`commit=, build date=, build source=homebrew, version=0.42.0, os=darwin, arch=arm64, git version=2.45.2`

Please let me know if you need more info.

**Comment by stefanhaller** [comment]
Ah, thanks. With that recipe it is easy to reproduce. Here's a fix: #3660
